examples/stateful_example.yaml

```
@@ -0,0 +1,115 @@
+  level: info
+  format: plain
+
+# Global state management configuration
+state_management:
+  enabled: true
+  backend_type: memory  # Options: memory, s3, hybrid
+  checkpoint_interval_ms: 30000  # 30 seconds
+  retained_checkpoints: 3
+  exactly_once: true
+  state_timeout_ms: 3600000  # 1 hour
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)